test/parallel/test-util-types.js

```diff
@@ -68,6 +69,15 @@ for (const [ value, _method ] of [
   }
 }
 
+// Check boxed primitives.
```

Comment:
Maybe add some negative results to the test?

Reply:
This seems somewhat redundant to me, since this is just a helper combined out of different helpers that are already tested against all other types.
But I'll add negative results as well if you feel strongly about it!

Reply:
I think it makes sense to test - but don't feel strongly about it.